docs/src/main/asciidoc/telemetry-opentracing-to-otel-tutorial.adoc

```diff
@@ -0,0 +1,504 @@
+
+=== Generate the legacy project
+
+Create the legacy project by using the https://code.quarkus.io/[Quarkus code generator] with the `resteasy-reactive` and `quarkus-smallrye-opentracing` extensions, or by executing the following command:
+
+:create-app-artifact-id: opentracing-quickstart
+:create-app-extensions: resteasy-reactive,quarkus-smallrye-opentracing
+:create-app-code:
+include::{includes}/devtools/create-app.adoc[]
+
+This command generates the Maven project and imports the `smallrye-opentracing` extension, which
+includes the OpenTracing support and the default https://www.jaegertracing.io/[Jaeger] tracer.
+
+=== The application
+
+The Quarkus project has a single endpoint and the related class will look like this:
+
+[source,java]
+----
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+@Path("/hello")
+public class GreetingResource {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        return "Hello from RESTEasy Reactive";
+    }
+}
+----
+
+There is no OpenTracing specific code in the generated project, but the `smallrye-opentracing` extension is present and enabled by default, and it will automatically instrument the code.
+
+Let's start the Jaeger-all-in-one Docker image, where we will retrieve and see the captured traces:
+
+[source,bash]
+----
+docker run -e COLLECTOR_OTLP_ENABLED=true -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 4317:4317 -p 4318:4318 -p 14250:14250 -p 14268:14268 -p 14269:14269 -p 9411:9411 jaegertracing/all-in-one:latest
+----
+
+At this point you can run the application with Quarkus dev mode:
+
+include::{includes}/devtools/dev.adoc[]
+
+If you call the http://localhost:8080/hello[`/hello` endpoint] the related traces can be retrieved in the Jaeger UI at this address: http://localhost:16686/
+
+They will look like this:
+
+image::ot-to-otel-1.png[alt=OpenTracing span,role="center"]
+
+== Big bang change from OpenTracing to OpenTelemetry
+
+This is the happiest path, in this case there is no manual instrumentation. We can do a big bang change from OpenTracing to OpenTelemetry without side effects.
+
+=== Change dependencies
+
+To migrate between the two frameworks, you must drop the old `quarkus-smallrye-opentracing` extension and replace it by the new the `quarkus-opentelemetry` extension in the pom.xml file:
+
+The legacy extension is removed from the project:
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-smallrye-opentracing</artifactId>
+</dependency>
+----
+
+The new one is added:
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-opentelemetry</artifactId>
+</dependency>
+----
```

Comment:
Could you use the tabs here? See example here: https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/validation.adoc?plain=1#L57-L70

Reply:
done